src/theme/components/notification.ts

```diff
@@ -62,6 +65,10 @@ const NotificationContent = {
         ? "calc(var(--nypl-space-m) + var(--nypl-space-s))"
         : null,
       width: "100%",
+      // Links should always be black and underlined.
+      a: {
+        color: "ui.black",
```

Comment:
I did not state this initially (truthfully, I am just thinking of it now), but the `hover` state also needs to be black.  There should be no color change, ever.  We don’t have `visited` or `active` colors, so it is only `hover` that we need to worry about.